train.py

```diff
@@ -54,7 +54,7 @@ def get_base_argument_parser():
     # logging
     parser.add_argument("--experiment_name", type=str, default="dev")
     parser.add_argument("--logs_path", type=str, default="log")
-    parser.add_argument("--filepath_to_arguments_json_config", type=str, default=None)
+    parser.add_argument("-config", "--filepath_to_arguments_config", type=str, default=None)
```

Comment:
This is fair, but I have a hunch that some of the code I wrote that takes care of argument parsing does not cover shortcuts. I would be OK with you just changing the default to config. We can do this properly with Hydra, later.

Reply:
Ah ok. 

I like the descriptiveness of the longer name, I just don't always want to type it out. I think what I will do is change the long one and add a `help` hint.

Reply:
Sounds good. :)